### PR TITLE
ALS-1377 Adding new line when removing declared node

### PR DESCRIPTION
**Pull request by emilianoascona**

### CODE REVIEW

**Review by agusayerza — APPROVED** [review]

**Review by llibarona — CHANGES_REQUESTED** [review]
why is the golden diff showing `+\n\n`?
should de difference not be only one EOL?

**Review by agusayerza — CHANGES_REQUESTED** [review]
Fix \n\n

File: als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsHandler.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package org.mulesoft.als.server.modules.actions.codeactions
+
```
Comment (llibarona) [inline als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsHandler.scala:2]: was this added on purpose?

File: als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsManager.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.mulesoft.als.server.modules.actions.codeactions
+
+import org.mulesoft.als.server.logger.Logger
+import org.mulesoft.als.server.workspace.WorkspaceManager
+import org.mulesoft.lsp.feature.telemetry.TelemetryProvider
+
+class CodeActionsManager(val workspace: WorkspaceManager,
+                         private val telemetryProvider: TelemetryProvider,
```
Comment (llibarona) [inline als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsManager.scala:8]: was this added on purpose?

File: als-server/shared/src/test/scala/org/mulesoft/als/server/workspace/codeactions/DeleteNodeActionTest.scala [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        |
+        |uses:
+        |    lib: l.raml""".stripMargin,
+    "file:///root/l.raml" ->
+      """#%RAML 1.0 Library
+        |types:
+        |    t2:  | number""".stripMargin
+  )
+
+  val te1: TestEntry = TestEntry("file:///root/l.raml",
```
Comment (llibarona) [inline als-server/shared/src/test/scala/org/mulesoft/als/server/workspace/codeactions/DeleteNodeActionTest.scala:88]: first parameter is named "deletedElement", why is there an uri here?

**Review by llibarona — CHANGES_REQUESTED** [review]

File: als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsHandler.scala (hunk 1 above)
Comment (emilianoascona) [reply als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsHandler.scala:2]: Yes, I forgot to delete it and didn't see it in the files.

File: als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsManager.scala (hunk 2 above)
Comment (emilianoascona) [reply als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsManager.scala:8]: Same as in CodeActionsHandler

**Review by agusayerza — APPROVED** [review]

**Review by agusayerza — APPROVED** [review]

**Review by llibarona — APPROVED** [review]

**Pull request merged**